### ISSUE TITLE
Problem with sourceJar after upgrade to 0.31.1

### ISSUE THREAD

**Issue by PierreLeresteux** [kordamp/kordamp-gradle-plugins#235]
Hello,

since the 0.31.1 version, my Kotlin multi-project won't compile.

It will throws me a NPE

`Caused by: java.lang.NullPointerException
	at org.kordamp.gradle.plugin.source.SourceJarPlugin.createSourceJarTask(SourceJarPlugin.groovy:136)`

To compile and investigate, I disable the `config.source` (because of code I read it : https://github.com/kordamp/kordamp-gradle-plugins/commit/f6da10359ff6ffbc0b1a28b431e3019dc93c0f2b#diff-972e9b79fba737cf3ab38e2f653d3ed4R136) but no source artifact are created (so I cannot publish it on bintray).

In 0.30.4 (and config.source.enabled=true) :
```
> Task :core:mainPublicationSettings
main:
    type: Maven
    groupId: io.saagie
    artifactId: core
    version: 0.0.1
    artifacts:
        artifact 0:
            file: core/build/libs/core-0.0.1-javadoc.jar
            classifier: javadoc
            extension: jar
        artifact 1:
            file: core/build/libs/core-0.0.1-sources.jar
            classifier: sources
            extension: jar
        artifact 2:
            file: core/build/libs/core-0.0.1.jar
            extension: jar
```
In 0.31.1 (and config.source.enabled=false): 
```
> Task :core:mainPublicationSettings
main:
    type: Maven
    groupId: io.saagie
    artifactId: core
    version: 0.0.1
    artifacts:
        artifact 0:
            file: core/build/libs/core-0.0.1.jar
            extension: jar
```
In 0.31.1 (and config.source.enabled=true):
`NullPointerException`

 
I create a branch in my project : https://github.com/PierreLeresteux/magicalupdater/pull/1/files

I don't know if this problem is from this gradle-plugin or a configuration in my own project ... 

PS : BTW, Happy new year ! And thanks for this plugin ! It helps me a lot !

Thanks.

**Comment by aalmiray** [comment]
Hi Pierre, yes, this is a bug in `0.31.1` and a fix is underway. A fix release (`0.31.2`) will be available this weekend. Sorry for the temporary inconvenience.

**Comment by aalmiray** [comment]
FYI make sure you apply the `org.kordamp.gradle-kotlin-project` to the root instead of `org.kordamp.gradle.project`, as the former adds more features that are Kotlin specific.
